BLUESPAWN-client/src/util/permissions/permissions.cpp

```diff
@@ -163,6 +164,43 @@ namespace Permissions {
 	PSID SecurityDescriptor::GetUserSID() const { return this->lpUserSID; }
 	PSID SecurityDescriptor::GetGroupSID() const { return this->lpGroupSID; }
 
+	bool Owner::bPolicyInitialized{ false };
+	LsaHandleWrapper Owner::lPolicyHandle{ nullptr };
+	const std::vector<std::wstring> Owner::vSuperUserPrivs{ SE_DEBUG_NAME, SE_IMPERSONATE_NAME, SE_TCB_NAME, SE_LOAD_DRIVER_NAME,
+			SE_ASSIGNPRIMARYTOKEN_NAME, SE_TAKE_OWNERSHIP_NAME };
+
+	void LsaHandleWrapper::SafeCloseLsaHandle(LSA_HANDLE handle) {
+		LsaClose(handle);
+		Owner::bPolicyInitialized = false;
+	}
+
+	void Owner::InitializePolicy() {
+		LSA_OBJECT_ATTRIBUTES lObjectAttr;
+		memset(&lObjectAttr, 0, sizeof(LSA_OBJECT_ATTRIBUTES));
+		HRESULT hr = LsaNtStatusToWinError(LsaOpenPolicy(nullptr, &lObjectAttr, GENERIC_ALL, &lPolicyHandle));
+		if (hr != ERROR_SUCCESS) {
+			bPolicyInitialized = false;
+			LOG_ERROR("Couldn't open policy handle. (Error:" << hr << ")");
+			SetLastError(hr);
+		}
+		else {
+			bPolicyInitialized = true;
+		}
+	}
+
+	LSA_UNICODE_STRING Owner::WStringToLsaUnicodeString(IN const std::wstring& str) {
+		LSA_UNICODE_STRING lsaWStr;
```

Comment:
Always initialize variables when defining them. This should be LSA_UNICODE_STRING lsaWStr{};